code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -106,6 +106,10 @@
 				R.hands.icon_state = module_holder
 
 /obj/item/weapon/robot_module/proc/AddCameraNetworks(var/mob/living/silicon/robot/R)
+	if(!R.camera && networks.len > 0) //Alright this module adds the borg to a CAMERANET but it has no camera, so we give it one.
```

Comment:
🤔 
are you absolutely sure this wont result in unintended behavior

Reply:
Soviet MoMMIs have no network list, NT MoMMIs have CAMERANET_ENGI